### PR TITLE
incus/project: Fix handling of default (unset) project in `get-current`

### LINKED ISSUE #1474: project: `incus project get-current` is empty if you never switched the project
# Required information

 * Distribution: Gentoo
 * Distribution version: --
 * The output of "incus info" or if that fails:
   * Incus version: 6.7

# Issue description

The command `incus project get-current` should write the name of the current project to stdout which is the `default` project initially.

However, if you _never_ used `incus project switch` command to switch to some project and then back to default, then the output of `get-current` is empty instead of `default`

# Steps to reproduce

 1. fresh installation
 2. call `incus project get-current` -> no  output!
 3. create some new project: `incus project create myproject`
 4. switch: `incus project switch myproject`
 5. call `incus project get-current` ->  "myproject"
 6. switch back: `incus project switch default`
 7. call `incus project get-current` ->  "default"

### PR DESCRIPTION
**Pull request by irhndt**
Closes #1474 

**First contribution to opensource sorry if I missed some of the best practices**

### CODE REVIEW

File: cmd/incus/project.go [hunk 1]
```diff
@@ -1184,7 +1184,10 @@ func (c *cmdProjectGetCurrent) Run(cmd *cobra.Command, args []string) error {
 	if !ok {
 		return fmt.Errorf(i18n.G("Remote %s doesn't exist"), conf.DefaultRemote)
 	}
-
-	fmt.Println(remote.Project)
+	if remote.Project == "" {
+		fmt.Println("default")
```
Comment (stgraber) [inline cmd/incus/project.go:1188]: Instead of passing the `"default"` string, it'd be preferable to use `api.ProjectDefaultName`.

Comment (irhndt) [reply cmd/incus/project.go]: Thanks for the notice, added a commit fixing it the good way.

**Comment by stgraber** [comment]
@irhndt looks good. I did a quick rebase for you and just retitled the commit.

The one thing we're missing before this can be merged is a suitable `Signed-off-by` line in the commit message. You can see CONTRIBUTING.md for this part.

**Comment by irhndt** [comment]
Hi, signed the commit should good.

**Review by stgraber — APPROVED** [review]

**Pull request merged**